crypto/dilithium/pqcrystals_dilithium_ref_common/rounding.c

```diff
@@ -18,8 +18,8 @@
 int32_t power2round(int32_t *a0, int32_t a)  {
   int32_t a1;
 
-  a1 = (a + (1 << (D-1)) - 1) >> D;
-  *a0 = a - (a1 << D);
+  a1 = (a + (1 << (ML_DSA_D-1)) - 1) >> ML_DSA_D;
```

Comment:
Please update rounding.h

Reply:
done in [2bbdfc9](https://github.com/aws/aws-lc/pull/2072/commits/2bbdfc975539e79985135f515caa45399229d1f6) (I also missed `packing.h`